samples/multiplatform/src/jvmTest/junit5/src/test/kotlin/ch/tutteli/SmokeTest.kt

```diff
@@ -0,0 +1,66 @@
+package ch.tutteli
+
+import org.junit.jupiter.api.*;
+
+import ch.tutteli.atrium.api.fluent.en_GB.*
+import ch.tutteli.atrium.api.verbs.assertThat
+import ch.tutteli.atrium.assertions.Assertion
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.AssertImpl
+import ch.tutteli.atrium.reporting.RawString
+import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
+import ch.tutteli.atrium.translations.DescriptionBasic
+
+class SmokeTest {
+    @Test
+    fun toBe_canBeUsed() {
+        assertThat(1).isLessThan(2)
```

Comment:
change to 
```suggestion
        assertThat(1).toBe(2) 
```
so that you see the test fails in the build